include/deal.II/base/tensor.h

```diff
@@ -359,6 +365,17 @@ class Tensor<0, dim, Number>
   constexpr DEAL_II_CUDA_HOST_DEV real_type
   norm_square() const;
 
+  /**
+   * Fill a range with all tensor elements. Since this type of Tensor only has
+   * one entry this just copies the value of this tenso into <tt>*begin</tt>.
+   *
+   * The template type Number must be convertible to the type of
+   * <tt>*begin</tt>.
+   */
+  template <class Iterator>
+  void
+  unroll(const Iterator begin, const Iterator end) const;
```

Comment:
How much do you want to change the name? I like `unroll()` because it matches the semantics of the two indexing functions and the type signature already implies that it doesn't change the Tensor.

Reply:
@bangerth not urgent but let me know when you have time to take another look - I think this is ready once we agree on a name.